packages/css/src/transformCss.test.ts

```diff
@@ -548,6 +548,219 @@ describe('transformCss', () => {
+    expect(
+      transformCss({
+        composedClassLists: [],
+        localClassNames: ['testClass'],
+        cssObjs: [
+          {
+            type: 'local',
+            selector: 'testClass',
+            rule: {
+              content: "'hello there'",
+              selectors: {
+                '&': { content: '"hello there"' },
+              },
+            },
+          },
+        ],
+      }).join('\n'),
+    ).toMatchInlineSnapshot(`
+      ".testClass {
+        content: 'hello there';
+      }
+      .testClass {
+        content: \\"hello there\\";
+      }"
+    `);
+  });
+
+  it('should not add quotes to meaningful content values (examples from mdn: https://developer.mozilla.org/en-US/docs/Web/CSS/content)', () => {
+    expect(
+      transformCss({
+        composedClassLists: [],
+        localClassNames: ['testClass'],
+        cssObjs: [
+          {
+            type: 'local',
+            selector: 'testClass',
+            rule: {
+              content: 'normal',
+              selectors: {
+                '._01 &': { content: 'none' },
+                '._02 &': { content: 'url("http://www.example.com/test.png")' },
+                '._03 &': { content: 'linear-gradient(#e66465, #9198e5)' },
+                '._04 &': {
+                  content: 'image-set("image1x.png" 1x, "image2x.png" 2x)',
+                },
+                '._05 &': {
+                  content:
+                    'url("http://www.example.com/test.png") / "This is the alt text"',
+                },
+                '._06 &': { content: '"prefix"' },
+                '._07 &': { content: 'counter(chapter_counter)' },
+                '._08 &': { content: 'counter(chapter_counter, upper-roman)' },
+                '._09 &': { content: 'counters(section_counter, ".")' },
+                '._10 &': {
+                  content:
+                    'counters(section_counter, ".", decimal-leading-zero)',
+                },
+                '._11 &': { content: 'attr(value string)' },
+                '._12 &': { content: 'open-quote' },
+                '._13 &': { content: 'close-quote' },
+                '._14 &': { content: 'no-open-quote' },
+                '._15 &': { content: 'no-close-quote' },
+                '._16 &': { content: 'open-quote counter(chapter_counter)' },
+                '._17 &': { content: 'inherit' },
+                '._18 &': { content: 'initial' },
+                '._19 &': { content: 'revert' },
+                '._20 &': { content: 'unset' },
+              },
+            },
+          },
+        ],
+      }).join('\n'),
+    ).toMatchInlineSnapshot(`
+      ".testClass {
+        content: normal;
+      }
+      ._01 .testClass {
+        content: none;
+      }
+      ._02 .testClass {
+        content: url(\\"http://www.example.com/test.png\\");
+      }
+      ._03 .testClass {
+        content: \\"linear-gradient(#e66465, #9198e5)\\";
```

Comment:
This this is a mis-quote 😝

Reply:
Pushed a fix. We didn't match on hyphens in function names.